PWGJE/Tasks/jetHadronRecoil.cxx

```diff
@@ -166,6 +170,13 @@ struct hJetAnalysis {
       registry.fill(HIST("hJetPt"), jet.pt());
       registry.fill(HIST("hJetEta"), jet.eta());
       registry.fill(HIST("hJetPhi"), jet.phi());
+      for (auto& jetWTA : jet.template matchedJetGeo_as<std::decay_t<W>>()) {
+        double deltaPhi = jetWTA.phi() - jet.phi();
```

Comment:
You should constrain the angle to be within +-Pi. Otherwise you'll get nonsensical values of `dR`.
See
https://github.com/AliceO2Group/O2Physics/blob/4ecf524826c7279cb172e35ac2b390f9696dd84c/Common/Core/RecoDecay.h#L214-L220

Reply:
I am not sure why this would ever be a problem, as I am only comparing geometrically matched jets. I do have a detected deltaPhi function that can be used if needed.

Reply:
Should be working now

Reply:
I need the second dPhi function for the delta phi observable which I want to be constrained between 0 and 2pi. I can change to use this constrainAngle function now though

Reply:
Should now be implemented